R/load.R

```diff
@@ -70,7 +70,7 @@ load <- function(project = NULL, quiet = FALSE, profile = NULL, ...) {
 
   action <- renv_load_action(project)
   if (action[[1L]] == "cancel") {
-    cancel()
+    quietly(cancel())
```

Comment:
Without this (but with the other changes in this PR), the user sees a potentially confusing message about cancelling an operation (what operation?) at startup:

```
- Operation canceled.
- Project '~/rrr/renv-coldstart' loaded. [renv 1.0.7.9000]
```

This is my best attempt to suppress this, using what I see here re: suppressing messages. I'd rather put the `quietly()` around `return("cancel")` below, but that doesn't actually suppress the message.

Reply:
I think this mostly comes down to whether the cancel was performed interactively (the user actually asked to cancel) versus us non-interactively cancelling.

Maybe we can detect if the autoloader is running at that point, and do something like `cancel(quiet = TRUE)`, and update the definition of `cancel` appropriately?

Reply:
OK good to know it's OK to propose a change in `cancel()`. I'll work on adding that to this PR.